### ISSUE TITLE
Failing to originate contract

### ISSUE THREAD

**Issue by emishur** [baking-bad/pytezos#132]
Pytezos version: 2.5.12

How to reproduce:
```python
from pytezos import pytezos
from pytezos import Contract

contract = Contract.from_file("./token_owner.tz")
pytezos.origination(script=contract.script()).autofill().sign().inject()
```
It seems does not work with any contract, but just in case here is the contract code I used:
[token_owner.tz](https://github.com/tqtezos/smart-contracts/blob/3fded8aeaabba6525741ce13e7f25ea282cbdfc7/multi_asset/ligo/out/token_owner.tz)

Error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/tools/docstring.py", line 67, in __call__
    return method(self.class_instance, *args, **kwargs)
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/operation/group.py", line 176, in autofill
    opg_with_metadata = opg.run()
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/tools/docstring.py", line 67, in __call__
    return method(self.class_instance, *args, **kwargs)
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/operation/group.py", line 140, in run
    return self.shell.blocks[block_id].helpers.scripts.run_operation.post({
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/tools/docstring.py", line 67, in __call__
    return method(self.class_instance, *args, **kwargs)
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/rpc/helpers.py", line 248, in post
    return self._post(json=operation)
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/rpc/query.py", line 136, in _post
    return self.node.post(
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/rpc/node.py", line 118, in post
    response = self.request('POST', path, params=params, json=json)
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/rpc/node.py", line 92, in request
    raise RpcError.from_response(res) from None
pytezos.rpc.node.RpcError: ('Failed to parse the request body: No case matched:\n'
 '  At /kind, unexpected string instead of endorsement\n'
 '  At /kind, unexpected string instead of seed_nonce_revelation\n'
 '  At /kind, unexpected string instead of double_endorsement_evidence\n'
 '  At /kind, unexpected string instead of double_baking_evidence\n'
 '  At /kind, unexpected string instead of activate_account\n'
 '  At /kind, unexpected string instead of proposals\n'
 '  At /kind, unexpected string instead of ballot\n'
 '  At /kind, unexpected string instead of reveal\n'
 '  At /kind, unexpected string instead of transaction\n'
 '  Unhandled error (Invalid_argument "Z.of_substring_base: invalid digit")\n'
 '  At /kind, unexpected string instead of delegation',)
```

**Comment by m-kus** [comment]
Can you pls attach the value returned by `sign()` (before injecting)?

**Comment by emishur** [comment]
`sign()` fails with the same error:

```
pytezos.origination(script=contract.script()).autofill().sign()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/tools/docstring.py", line 67, in __call__
    return method(self.class_instance, *args, **kwargs)
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/operation/group.py", line 176, in autofill
    opg_with_metadata = opg.run()
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/tools/docstring.py", line 67, in __call__
    return method(self.class_instance, *args, **kwargs)
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/operation/group.py", line 140, in run
    return self.shell.blocks[block_id].helpers.scripts.run_operation.post({
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/tools/docstring.py", line 67, in __call__
    return method(self.class_instance, *args, **kwargs)
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/rpc/helpers.py", line 248, in post
    return self._post(json=operation)
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/rpc/query.py", line 136, in _post
    return self.node.post(
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/rpc/node.py", line 118, in post
    response = self.request('POST', path, params=params, json=json)
  File "/Users/eugenemishura/opt/smart-contracts/.venv/lib/python3.9/site-packages/pytezos/rpc/node.py", line 92, in request
    raise RpcError.from_response(res) from None
pytezos.rpc.node.RpcError: ('Failed to parse the request body: No case matched:\n'
 '  At /kind, unexpected string instead of endorsement\n'
 '  At /kind, unexpected string instead of seed_nonce_revelation\n'
 '  At /kind, unexpected string instead of double_endorsement_evidence\n'
 '  At /kind, unexpected string instead of double_baking_evidence\n'
 '  At /kind, unexpected string instead of activate_account\n'
 '  At /kind, unexpected string instead of proposals\n'
 '  At /kind, unexpected string instead of ballot\n'
 '  At /kind, unexpected string instead of reveal\n'
 '  At /kind, unexpected string instead of transaction\n'
 '  Unhandled error (Invalid_argument "Z.of_substring_base: invalid digit")\n'
 '  At /kind, unexpected string instead of delegation',)
```

**Comment by m-kus** [comment]
Was able to reproduce on 2.5.12, fixed in 2.5.13

**Comment by emishur** [comment]
resolved with 2.5.13
